src/main/kotlin/lotto/domain/RandomNumberGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+class RandomNumberGenerator : LottoNumberGenerator {
+    override fun generate(): List<LottoNumber> {
+        return (LottoNumber.MINIMUM_LOTTO_NUMBER..LottoNumber.MAXIMUM_LOTTO_NUMBER).toList().shuffled()
```

Comment:
```suggestion
        return (LottoNumber.MINIMUM_LOTTO_NUMBER..LottoNumber.MAXIMUM_LOTTO_NUMBER).shuffled()
```
방어적 복사를 활용을 잘 해주셨네요. 👍 
하지만, 대부분의 컬렉션 관련 함수들은 원본을 수정하지 않고 복사본을 만들어서 변경하곤 합니다.
그래서 toList()를 호출하지 않아도 됩니다.

shuffled() 함수의 내부 구현체는 아래와 같습니다. (toMutableList())
```kotlin
@SinceKotlin("1.2")
public actual fun <T> Iterable<T>.shuffled(): List<T> = toMutableList().apply { shuffle() }
```

Reply:
헉.. 내부 구조를 확인하지 않고 코드를 작성했더니..! 
말씀해준신대로 코드를 수정했습니다. 😅